### DISCUSSION TITLE
I still get the warning message about Transform API on newrelic v7.2

### DISCUSSION THREAD

**Discussion by lamphama2** [newrelic/newrelic-android-agent#167] [Q&A] [answered]
Hello,
From what I understand, the version **v7.2** is supposed to work with AGP 8.0 which means the Transform API is removed.
However, I still received the warning after bumping the version to **v7.2**.

```
WARNING:API 'android.registerTransform' is obsolete.
It will be removed in version 8.0 of the Android Gradle plugin.
The Transform API is removed to improve build performance. Projects that use the
Transform API force the Android Gradle plugin to use a less optimized flow for the
build that can result in large regressions in build times. It’s also difficult to
use the Transform API and combine it with other Gradle features; the replacement
APIs aim to make it easier to extend the build without introducing performance or
correctness issues.

There is no single replacement for the Transform API—there are new, targeted
APIs for each use case. All the replacement APIs are in the
`androidComponents {}` block.
...
```
Did I miss anything?

### Context
- AGP 7.4
- Gradle 7.5

**Comment by lamphama2** [comment]
From what I see, the `NewRelicTransform` class which extends `Transform` is still there which will breaks when we upgrade to AGP 8.0

**Reply by ywang-nr** [reply, marked as answer]
@lamphama2 This part of the code will not be removed due to backwards compatibility, your project should use the code corresponding to AGP8 after upgrading.

**Reply by lamphama2** [reply]
hey sorry it's actually a mistake.
I didn't correctly pump AGP to 7.5.
The second attempt worked as expected.
I'm resolving this.
